metadata_layer.py

```diff
@@ -361,6 +368,46 @@ def __init__(self, content_type, drm_scheme, link=None):
             )
         self.link = link
 
+
+class RecommendationData(object):
+
+    def __init__(self, data_source, identifiers=None):
+        self.data_source = data_source
+        self.identifiers = identifiers or []
+
+    @property
+    def recommended_works(self):
```

Comment:
Then you've got this thing which runs a query, and really ought to be in model.py. It feels like a method called `Work.from_identifiers`, which takes a list of Identifier objects and returns a list of Works.